### PR TITLE
Automate publishing

### PR DESCRIPTION
**Pull request by cloudnautique**
The PR will automate the following when a tag is placed on Master.

This will create the GH release if it doesn't exist yet. 
This will publish binaries from the ./bin/* directory to the GH release.
This will push a rancher/rke-certs-deployer:${tag} image to docker hub.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -7,3 +7,44 @@ pipeline:
...
+    - dapper build
+    when:
+      branch: master
+      event: tag
+   
+
+  stage-binaries:
+    image: rancher/dapper:1.11.2
+    commands:
+    - cp -r ./bin/* ./package/
```
Comment (galal-hussein) [inline .drone.yml]: With CROSS=1 make will create the binaries in build/bin

File: .drone.yml [hunk 2]
```diff
@@ -7,3 +7,44 @@ pipeline:
...
+
+  github_binary_release:
+    image: plugins/github-release
+    files: bin/*
+    secrets: [github_token]
+    when:
+      branch: master
+      event: tag
+   
+  publish-rke-certs-deployer:
```
Comment (galal-hussein) [inline .drone.yml]: Regarding the other docker images, should they also be included as a part of publishing images to docker hub

Comment (galal-hussein) [reply .drone.yml]: nginx proxy name: rancher/rke-nginx-proxy:0.1.0
rancher cert deployer: rancher/rke-cert-deployer:0.1.0
service side kick: rancher/rke-service-sidekick:0.1.0

File: .drone.yml (hunk 1 above)
Comment (cloudnautique) [reply .drone.yml]: In the staging step, this just moves the binary for the Docker image. The `scripts/build` still build bin/* binaries, so I think this part is OK.

Comment (galal-hussein) [reply .drone.yml]: okay, thanks

**Pull request merged**